### ISSUE TITLE
training accuracy  64.42%?

### ISSUE THREAD

**Issue by eustcPL** [markdtw/vqa-winner-cvprw-2017#2]
you mean the accuracy is 64.42% when training or validating? Why 'sigmoid multi-label classifier is also implemented but I can't train based on that'?

Thank you in advance!

**Comment by markdtw** [comment]
1. it's the training accuracy.

2. sigmoid classifier is hard to train in my experience and I didn't initialize the classifier's weights as paper suggests so maybe that's why.

Please note that the training accuracy is quite a bad report actually. If the learning rate is larger the training accuracy would be so much higher, validation accuracy should be reported as well and I should do that in the near future.

**Comment by ObadaJabassini** [comment]
Any updates?
